ttnn/ttnn/operations/core.py

```diff
@@ -466,41 +466,30 @@ def to_memory_config(tensor, memory_config: ttnn.MemoryConfig):
     """
 
     ttl_tensor = tensor.value
+    if tensor.memory_config == memory_config:
```

Comment:
might need to rebase its now ttnn.get_memory_config(tensor)

Reply:
Currently cant rebase to main due to some recent changes that require another change to go to main first before we can rebase 🙃 
Will update after